plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -1102,7 +1128,47 @@ actions:
         description: Was the operation successful
         type: boolean
         required: true
-
+  get_xql_query_results:
+    title: Get XQL Query Results
+    description: Start an XQL query and retrieve the query results
+    input:
+      query:
+        title: Query
+        description: String of the XQL query
+        type: string
+        required: true
+        example: dataset=xdr_data | fields event_id, event_type, event_sub_type | limit 3
+      tenants:
+        title: Tenants
+        description: String that represents additional information regarding the action
+        type: "[]string"
+        required: true
+        example: ["tenantID", "tenantID"]
+      to:
```

Comment:
Change to and from to `startTime` `endTime` perhaps. To and from can get a little confusing